tests/typescript/src/generators.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * @license
+ * Copyright 2024 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as Blockly from 'blockly-test/core';
+import {JavascriptGenerator} from 'blockly-test/javascript';
+import {PhpGenerator, phpGenerator, Order} from 'blockly-test/php';
+import {LuaGenerator} from 'blockly-test/lua';
+import {PythonGenerator} from 'blockly-test/python';
+import {DartGenerator} from 'blockly-test/dart';
+
+JavascriptGenerator;
+PhpGenerator;
+LuaGenerator;
+PythonGenerator;
+DartGenerator;
```

Comment:
Does this actually test something that the import doesn't?

Reply:
It prevents my editor and possibly eslint from complaining to me that the values are unused.

Reply:
I'm surprised the eslint doesn't complain about the pointlesness of these statements, but whatever…